### DISCUSSION TITLE
Can I switch to use_phonemes = true while training ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1233] [General Q&A] [answered]
Hi,

I have trained a VITS model from scratch on own French dataset (single voice) and after 50k the alignments look good (diagonals) but in audio test sentences words are minced (spoken very quickly). Would I get better results switching to phonemes training ("use_phonemes") when continuing the training ?

Thanks

**Comment by erogol** [comment, marked as answer]
You can but not suggested as you make it True, the model does not continue its learning but starts to learn almost from scratch.

**Reply by Ca-ressemble-a-du-fake** [reply]
Thanks @erogol . While reading the paper again I noticed that with VITS there is no need of using phonemes as `[...], unlike previous work [4, 19], in our model we used raw text as input instead of phonemes`. Now I understand why all config.json I see have `use_phonemes = False`!

**Comment by dostee-hemin** [comment]
Hello @Ca-ressemble-a-du-fake , I seem to be facing the same issue where the "audio test sentences words are minced (spoken very quickly)." on my VITS model. How did you manage to solve this problem? Did you just need more training or was something wrong with the configuration?

**Reply by Ca-ressemble-a-du-fake** [reply]
Hi @dostee-hemin, I started over from scratch with use phonemes = true and use espeak phonemes = true also. At about 20-30k steps the result was already promising. Now at about 70k steps it is amazing. 
However I increased the batch size from 20 to 64 which should have helped a lot (and switch to another GPU). I am in the process of writing a step by step guide on the exact methodology I followed. As soon as it is published I will post a piece of news here!

**Reply by dostee-hemin** [reply]
Thank you so much for your insight, I'll give it a shot and see what happens. Wish you the best of luck on your project!
